src/main/kotlin/no/nav/familie/pdf/pdf/XmpMeta.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.familie.pdf.pdf
+
+import com.itextpdf.kernel.xmp.XMPMeta
+import com.itextpdf.kernel.xmp.XMPMetaFactory
+
```

Comment:
Skulle gjerne hatt litt dokumentasjon i denne filen som sier litt om hva dette her er for noe :)

Reply:
gotcha ;)